### ISSUE TITLE
Running test_core.py problem with dlpath

### ISSUE THREAD

**Issue by FacundoGFlores** [JuliaInterop/pyjulia#21]
I installed julia from the ubuntu repository, and I also tested compiling the version 0.3, and it still does not work. 

I'm trying to run the program "test_core.py" and I get:

```bash
python test_core.py 
ERROR: dlpath not defined
 in process_options at client.jl:242
 in _start at client.jl:389
Traceback (most recent call last):
  File "test_core.py", line 7, in <module>
    julia = Julia()
  File "/usr/local/lib/python2.7/dist-packages/julia/core.py", line 228, in __init__
    raise JuliaError('error starting up the Julia process')
julia.core.JuliaError: error starting up the Julia process
```

The same error appears when I just try (inside python) 

```python
import julia
julia.Julia()
```

**Comment by jacksonloper** [comment]
Sys.dlpath appears to have been deprecated in Julia 0.5 in favor of Libld.dlpath

https://github.com/JuliaLang/julia/pull/10328

**Comment by Keno** [comment]
Fixed by #54.
